docs/Documentation/UserGuideV0.7.0/7-TsFile/2-Usage.md

```diff
@@ -0,0 +1,765 @@
+	public QueryDataSet query(QueryExpression queryExpression) throws IOException
+	```
+
+* **Method 2**
+
+	```
+	public QueryDataSet query(QueryExpression queryExpression, long partitionStartOffset, long partitionEndOffset) throws IOException
+	```
+
+	This method is designed for advanced applications such as the TsFile-Spark Connector.
+
+	* **params** : For method 2, two additional parameters are added to support partial query:
+		*  ```partitionStartOffset```: start offset for a TsFile
+		*  ```partitionEndOffset```: end offset for a TsFile
+
+		> **What is Partial Query ?**
+		>
+		> In some distributed file systems(e.g. HDFS), a file is split into severval parts which are called "Blocks" and stored in different nodes. Executing a query paralleled in each nodes involved makes better efficiency. Thus Partial Query is needed. Paritial Query only selects the results stored in the part split by ```QueryConstant.PARTITION_START_OFFSET``` and ```QueryConstant.PARTITION_END_OFFSET``` for a TsFile.
+
+#### QueryDataset Interface
+
+The query performed above will return a `QueryDataset` object.
+
+Here's the useful interfaces for user.
+
+
+* `bool hasNext();`
+
+    Return true if this dataset still has elements.
+* `List<Path> getPaths()`
+
+    Get the paths in this data set.
+* `List<TSDataType> getDataTypes();` 
+
+   Get the data types. The class TSDataType is an enum class, the value will be one of the following:
+   
+       BOOLEAN,
+       INT32,
+       INT64,
+       FLOAT,
+       DOUBLE,
+       TEXT;
+ * `RowRecord next() throws IOException;`
+ 
+    Get the next record.
+    
+    The class `RowRecord` consists of a `long` timestamp and a `List<Field>` for data in different sensors,
+     we can use two getter methods to get them.
+     
+    ```
+    long getTimestamp();
+    List<Field> getFields();
+    ```
+    
+    To get data from one Field, use these method:
+    
+    ```
+    TSDataType getDataType();
+    Object getObjectValue();
+    ```
+
+#### Example for reading an existing TsFile
+
+You should install TsFile to your local maven repository.
+
+See reference: [Installation](./1-Installation.md)
+
+
+```java
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
```

Comment:
license is not needed here

Reply:
I saw there were license statements in other .md files.